### PR TITLE
community/caddy: upgrade to 0.11.4

### PR DESCRIPTION
**Pull request by TBK**
https://github.com/mholt/caddy/releases/tag/v0.11.4

### CODE REVIEW

**Comment by andypost** [comment]
@clandmeter it works fine locally

**Comment by TBK** [comment]
Bumped to 0.11.4 which was released yesterday.

File: community/caddy/APKBUILD [hunk 1]
```diff
@@ -29,22 +29,18 @@ prepare() {
...
 build() {
-	cd "$builddir"
 	export GOPATH="$srcdir"
 	go build -v -o bin/caddy -ldflags \
 		"-X github.com/mholt/caddy/caddy/caddymain.gitTag=$pkgver" \
 		./caddy
 }
 
 check() {
-	cd "$builddir"
```
Comment (andypost) [inline community/caddy/APKBUILD]: please keep it to minify changes on backports

Comment (TBK) [reply community/caddy/APKBUILD]: Reverted.

**Pull request closed without merging**